src/api/dev.ts

```diff
@@ -57,6 +59,7 @@ class Watcher extends EventEmitter {
 		'dev-port': dev_port,
 		live,
 		hot,
+		'devtools-port': devtools_port,
```

Comment:
I've added this here with an eye towards making the DevTools port a command-line switch in the future.